service/controller/app/key/key.go

```diff
@@ -137,6 +138,18 @@ func KubecConfigSecretNamespace(customResource v1alpha1.App) string {
 	return customResource.Spec.KubeConfig.Secret.Namespace
 }
 
+func LabelSelectorService() labels.Selector {
```

Comment:
```suggestion
func AppLabelSelector() labels.Selector {
```

Do we need service in the name? What about this since its selecting apps?

Reply:
You're right, it's clear w/o service in the name.